### PR TITLE
A couple fixes and quality of life changes to Box Station.

### PR DESCRIPTION
**Pull request by zeroisthebiggay**
## About The Pull Request

Changes up some very minor some things in Box Station to make the map less inconvenient to play in general.

Breaths some new life into good 'ol Box Station to make it more up to par with the other maps in a few aspects related to convince and accessibility.

- The first of the three _major_ changes goes to the Cargo lobby, moving the ORM to the front in order to make it accessible to those with access (scientists, engineers wanting to get metal on the fly without running back to their department, etc) along with removing the old "autolathe line shutters" button that has served zero purpose other than remind us of the time public autolathes still were a thing.
2d2c321495da8064361054dd1248c6b4

- The second of our changes goes to the Vacant Commissary which has gotten a tiny makeover, nudging the tool storage room up slightly to make the emergency storage cubby holes more neat, noted on later, other than that though the floor tiles are more consistent, and there's a disposal bin in the primary hallway in the line to the commissary to promote employees not to litter.
Post Screenshot Note: I deleted the duplicate button and made the whole line in to the dark tiles.
dff9d25407563ecbf38ab15a452d5bc1

- After that, the arguably most significant change goes to the Cloning Lab, borrowing the exact same sort of design concepts from Delta Station and Meta Station featuring a small area for any freshly ejected clones (or plasmamen lacking their environment suits) to take a shower and fix up their hair styles without having to run all the way into the cryogenics bay to do so. I'm slightly iffy about this change as it's the only room I sort of _don't_ like so if you have any suggestions to make it better or if you simply don't like it then do please toss them my way.
60171968248b18e68c29cceb41410d39

- As for those minor changes, the two "emergency storage" rooms have had their fire lockers moved into the hallways so the station doesn't seem like it only has two fire lockers in the Science airlock, and also added a brand new fire locker to the escape wing. Put some emergency toolboxes in front of said emergency storage rooms for emergencies as their name implies.
- A few AI holopads have been put in places like primary tool storage and the cargo bay where they weren't before.
- Put some decals on things to make them look prettier.
- Anchored a singular oxygen locker in the AI sat so it doesn't blow itself out into deep space as soon as someone opens up the nearby airlock.
- Put two disposal bins in the central ring so Janitors and nonlitterers don't have to actively seek out a place to throw out their trash.
- Replaced the grey personal lockers in the Dormitories with the closets with wallets in them along with replacing their floors with wood _and_ carpet rather than _just_ carpet.
- Probably another thing or two i'm missing because this is mostly just a bunch of minor changes.
- Replaced a missing pipe in the atmospherics to incinerator loop and also made the plasma tank not export in to a wall.
- Renault exists.
- Runtime gets a pet bed and as does the brand new fox.

## Why It's Good For The Game

Box Station has been arguably one of the less popular maps along with Donut Station due to it's mentality of "old is good" which results in a majority of quality of life features being far too out of reach to be used regularly or simply nonexistant. I aim to make the map a little bit more convenient and updated so as to be more on par with the maps of current day so as to maybe one day see Box Station played and enjoyed as much as the likes of Meta Station. _(but we all know that's never happening lmao)_

## Changelog
:cl:
tweak: Nanotrasen is finally giving Box Station some love in the form of some minor tweaks to hopefully make a few aspects of the station more convenient after comparing it to some of the other models.
add: (BOX) Gave the Cargo Lobby, Vacant Commissary, and Cloning Lab a slight makeover to make them slightly nicer.
tweak: (BOX) Moved and added some fire lockers, moved some other things around a tad, put some disposal bins in the central ring, and finally anchored that ONE oxygen locker in the AI sat. You know the one I am talking about.
tweak: (BOX) A couple of other unnoteworthy quality of life changes.
del: (BOX) A singular metal chair in front of Cargo.
/:cl:

### CODE REVIEW

**Comment by AarontheIdiot** [comment]
i love you

**Review by actioninja — CHANGES_REQUESTED** [review]
run a find and replace on the text file of step_x to pixel_x and step_y to pixel_y. For future reference, do not use the nudge function in dreammaker it breaks map files, varedit the pixel var instead.
Alternatively use strongdmm.

**Comment by zeroisthebiggay** [comment]
> run a find and replace on the text file of step_x to pixel_x and step_y to pixel_y. For future reference, do not use the nudge function in dreammaker it breaks map files, varedit the pixel var instead.
> Alternatively use strongdmm.

big done.

**Comment by HomocidlePancakes** [comment]
Box is Box and it shouldn't be in line with Metastation based maps.

**Comment by MrFluffster** [comment]
Maps other than Meta definitely should get some more love

**Comment by HomocidlePancakes** [comment]
"love" isn't 100% the accurate term really. I just don't like that people are trying to shove the stuff from Meta in to every single map like it's some sort of golden standart that everyone should abide with. Like hell, we have Delta, which is basicly just a larger Meta. Two of the same map is enough, stop trying to make a third Meta clone.

**Comment by zeroisthebiggay** [comment]
> "love" isn't 100% the accurate term really. I just don't like that people are trying to shove the stuff from Meta in to every single map like it's some sort of golden standart that everyone should abide with. Like hell, we have Delta, which is basicly just a larger Meta. Two of the same map is enough, stop trying to make a third Meta clone.

The only things that have really been taken from Meta Station is the cloning lab's recovery room and the style of the cargo lobby.

And to your above point we all know box is box and we love it the way it is but everyone seems to really despise playing on any map that isn't Meta Station, especially with the map voting feature, as of the current page on statbus there's forty nine listings of Meta Station and a mere six of Box Station. I think the problem lies is that our beloved Meta Station has so many luxuries and convinces when it's compared to the rest of the maps in the line up that it's a pain to play anything but it so to make people want to play the other maps you either make /meta/ less of what it is or make /all the other maps/ more like it in their own ways.

tl;dr meta spoiled us.

**Comment by MrFluffster** [comment]
> "love" isn't 100% the accurate term really. I just don't like that people are trying to shove the stuff from Meta in to every single map like it's some sort of golden standart that everyone should abide with. Like hell, we have Delta, which is basicly just a larger Meta. Two of the same map is enough, stop trying to make a third Meta clone.

It just might be that Meta has some really good design features having had much much more improvements poured into it than any map.
Try to tell me how we can consider it remotely okay not to have the frickin' ORM not accessible to science when they and miners are the only ones ever using it.

**Comment by HomocidlePancakes** [comment]
>It just might be that Meta has some really good design features having had much much more improvements poured into it than any map.

I don't think Meta design choices are good, they exist just for the sake of appealing to the masses by making shit too accessiable. Delta has the same problem cranked up to 11. Basicly what the OP said "Meta spoiled us".

>Try to tell me how we can consider it remotely okay not to have the frickin' ORM not accessible to science when they and miners are the only ones ever using it.

"Hey CT, can you give me x sheets of plasma please?" or "AI please open". This was a somewhat valid argument when we didn't have the Ore Silo, but now I just can't help but laugh at you for bringing this up.

>The only things that have really been taken from Meta Station is the cloning lab's recovery room and the style of the cargo lobby.

I just fear that these changes will pile up over time. There already were PRs before to "Metanise" Box. The last one I can recall was about reinforcing brig just like on Meta.

>as of the current page on statbus there's forty nine listings of Meta Station and a mere six of Box Station

This isn't really a valid point because it's just how I feel, but I would like Box to show up less without Meta features in it, rather then it being more common, but with Meta stuff on it.

**Comment by MrPerson** [comment]
>"Hey CT, can you give me x sheets of plasma please?" or "AI please open". This was a somewhat valid argument when we didn't have the Ore Silo, but now I just can't help but laugh at you for bringing this up.

Ok, yes, it was possible to work around the issue. You've not explained why having the ORM not accessible is superior than having it accessible. It's just an obstacle to using it. There's no upside at all.

**Comment by vuonojenmustaturska** [comment]
Meta didn't spoil people by having too many resources, meta spoiled people by actually having a solid playable map with good fundamental design that extends to the maintenance. You could remove all the loose resources from meta and still be left with "a resource-scarce but fundamentally good map", whereas box's claim to fame seems to be "it used to be good compared to unnavigable stations made of wood and spiders." 

Box is almost 50% maintenance but almost all of it is awful, even cables and pipes break out of the station to travel through space rather than go through box's kleinbottle maintenance. Somehow maintenance only seems to connect to other areas of the same maintenance tunnel, or boxstation staples like "abandoned departmental atmos station", "a chair next to a window" or "a walled off empty locker/rack." On meta, you can use maintenance tunnels to move around on the station with regular breaks where you are forced briefly into a monitored area. On box, they're mostly disconnected individual segments full of hot garbage fluff rooms unfit for the purpose they're trying to establish.

tl;dr: box map bad

**Comment by HomocidlePancakes** [comment]
>Ok, yes, it was possible to work around the issue. You've not explained why having the ORM not accessible is superior than having it accessible. It's just an obstacle to using it. There's no upside at all.

Sure, I will explain to you once you explain to me why it’s in any way, shape or form a problem. Gonna repeat myself now. “This was a somewhat valid argument when we didn't have the Ore Silo, but now I just can't help but laugh at you for bringing this up.”

@vulnojrnmustaturska
1)You talk about how meta would be good even without all the loot on it and then spend most of your post saying how box is bad because it’s maint is empty. Self-contradiction.
2)As for the argument off “you can navigate the whole station on meta via maint”, you can do the same on box. Same low-view areas inbetween maints just like on meta too. The only exception is cargo, which forces you to either take the long and safe path to avoid it, or risk it breaking in to cargo/going through the halls. And by the way, meta also has a risky area like that inbetween lockers and cargo.

**Comment by MrFluffster** [comment]
> > Ok, yes, it was possible to work around the issue. You've not explained why having the ORM not accessible is superior than having it accessible. It's just an obstacle to using it. There's no upside at all.
> 
> Sure, I will explain to you once you explain to me why it’s in any way, shape or form a problem. Gonna repeat myself now. “This was a somewhat valid argument when we didn't have the Ore Silo, but now I just can't help but laugh at you for bringing this up.”


Scientists are the people who get materials from the ORM. On Meta they can do it. On Box they can't because it's behind access.



> @vuonojenmustaturska
> 1)You talk about how meta would be good even without all the loot on it and then spend most of your post saying how box is bad because it’s maint is empty. Self-contradiction.



He said that box is horribly designed, pointless labyrinthine and that even pipes and cables break the layout of maint(which if you didn't know is a big nono for any decent  mapper). The only point he mentioned loot is when saying that meta works well even without it.
Maybe if you spent the time you spent on typing out things like "I just can't help but laugh at you for bringing this up" on reading his post you'd realize that.

**Comment by HomocidlePancakes** [comment]
@MrFluffster 
When was the last time you played this game? Do you know what an ore silo is? Do you know that all departments have a departmental lathe that is connected to said silo, aswell as the ORM, which allows you to access mats without even leaving your department?

**Comment by MrFluffster** [comment]
> @MrFluffster
> When was the last time you played this game? Do you know what an ore silo is? Do you know that all departments have a departmental lathe that is connected to said silo, aswell as the ORM, which allows you to access mats without even leaving your department?


Yes, and?

**Comment by FloranOtten** [comment]
>I just fear that these changes will pile up over time. There already were PRs before to "Metanise" Box. The last one I can recall was about reinforcing brig just like on Meta.

Oh hey, that's my PR. I didn't make brig more reinforced, just the HoS office. I didn't do it because Meta has it that way, I did it because currently, Box's HoS locker can be stolen without any space protection. You can get it by just walking up, welding, falsewalling in and grabbing it.

**Comment by optimumtact** [comment]
Imagine thinking meta had a better map design when box was made by a map designer who made one of the best team fortress maps of all time

**Comment by zeroisthebiggay** [comment]
bruh i added one box.
